content/case_histories/balboa/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _balboa_index:
+
+The Balboa ZTEM Cu-Mo-Au porphyry discovery at Cobre Panama
+===========================================================
+
+.. .. raw:: html
+..     :file: ../../../underconstruction.html
+
+- **Authors**: Jean M. Legault, Chris Wijns, Carlos Izarra, and Geoffrey Plastow
```

Comment:
could you add their info to contributors.json? (even without pictures is fine). Thanks!